doc/nix-integration.rst

```diff
@@ -11,10 +11,8 @@ Nix Integration
 Enabling Nix Integration
 ------------------------
 
-To enable Nix integration, simply pass the ``--enable-nix`` global
-option when you call ``cabal``. To use this option everywhere, edit
-your :ref:`global configuration file<config-file-discovery>`
-(default: ``$HOME/.cabal/config``) to include:
+To enable Nix integration, simply pass the ``--enable-nix`` global option when you call ``cabal`` (eg. ``cabal --enable-nix build``).
```

Comment:
I'm afraid this won't work with new enough cabals, because now `build` is an alias for `v2-build`.  You probably want to say `v1-build`. Actually, if the flag does nothing with v2- commands, it would be great to also change all the examples below by prepending command names with `v1-`? Otherwise, we are just adding content to misleading documentation.